.env.local.default

```diff
@@ -15,7 +15,7 @@ KEY=certs/key.pem
 
 
 # Client variables ---------------
-APP_TITLE=Ethereal Engine
+APP_TITLE="Ethereal Engine"
```

Comment:
please revert these .env.local.default values

Reply:
I can't do so. It's not a valid bash env variable. I can also escape the space as an alternative

Reply:
Alright I have confirmed it does not change any exiting behavior